ansible/runload.yaml

```diff
@@ -4,19 +4,20 @@
   hosts: loadrunners
   vars:
     ansible_host_key_checking: false
-    endpoints: ws://127.0.0.1:26656/websocket # comma separated list of endpoints
+    validator_0_ip: "{{ groups['validators'][0] }}"
+    validator_0_internal_ip: "{{ hostvars[validator_0_ip].internal_ip }}"
+    endpoints: ws://{{ validator_0_internal_ip }}:26657/v1/websocket # comma separated list of endpoints
```

Comment:
... while this one can apparently use the `validator_0_internal_ip` variable directly?

Reply:
(see comment above)